artist/src/main/kotlin/com/uber/artist/ArtistExtension.kt

```diff
@@ -43,4 +43,8 @@ class ArtistExtension {
    */
   var formatSource: Boolean = true
 
+  /**
+   * Optional setting to control whether the source is generated in Java or Kotlin. Defaults to false for Java.
+   */
+  var generateKotlin: Boolean = false
```

Comment:
Would there ever be a case where someone wants to/can generate mixed sources? If so, should there also be a java flag?

Reply:
Hmm, yeah it might be nice to support that